vendor/forms.py

```diff
@@ -85,23 +85,39 @@ class AddressForm(forms.ModelForm):
 
     class Meta:
         model = Address
-        fields = ['name',  'first_name', 'last_name', 'country', 'address_1', 'address_2', 'locality', 'state', 'postal_code']
+        fields = ['name', 'first_name', 'last_name', 'country', 'address_1', 'address_2', 'locality', 'state', 'postal_code']
+
+    def __init__(self, *args, **kwargs):
+        super(AddressForm, self).__init__(*args, **kwargs)
+        self.fields['name'].hidden = True
+        self.fields['country'].label = _('Country/Region')
```

Comment:
Is there any issue in updating the Model to use these labels rather than redefining them in the Form?

Reply:
you are right gonna move them to the model